software-table.tex

```diff
@@ -47,6 +47,8 @@
     Optimizes parameters in models of demographic history using SFS and linkage disequilibrium statistics
     \citep{jouganous2017inferring,ragsdale2019models}. Models to be optimized can be specified in Demes.\\
 
+\MSMC & A script provided in the MSMC-tools repository (https://github.com/stschiff/msmc-tools) converts MSMC output to the demes format. Instructions are available in the Guide (https://github.com/stschiff/msmc-tools/blob/master/msmc-tutorial/guide.md)
```

Comment:
Can you use ``\url{https://...}`` for the URLs please?

Reply:
Done